NuRadioReco/modules/efieldToVoltageConverter.py

```diff
@@ -35,11 +34,12 @@ def __init__(self, log_level=logging.WARNING):
         self.__post_pulse_time = None
         self.__max_upsampling_factor = None
         self.antenna_provider = None
-        self.begin()
         self.logger = logging.getLogger('NuRadioReco.efieldToVoltageConverter')
         self.logger.setLevel(log_level)
+        self.begin()
+
 
-    def begin(self, debug=False, uncertainty=None,
+    def begin(self, debug=False, uncertainty={},
```

Comment:
The reason many default arguments in NuRadioMC are `None` rather than the object it defaults to is because of [this](https://stackoverflow.com/questions/41686829/why-does-pycharm-warn-about-mutable-default-arguments-how-can-i-work-around-the). 
While you're probably okay here (I don't think `uncertainty` is modified inside the begin function?) it's probably good practice not to use mutable objects as default arguments in general, so I'd suggest leaving it the way it was.

Reply:
Can we use or?

Reply:
Is that okay @sjoerd-bouma